README.md

```diff
@@ -17,6 +17,14 @@ Run the `make docs-serve` command in the root of the project, and then browse to
 
 After writing the documentation, please use the `make docs-vale` command and correct any warnings raised by the tool.
 
+## Local installation for testing purposes
+
+`make install` allows you to install the operator in a local (kind) cluster.
+
+This target installs the Prometheus Operator, according to the instructions explained [here](https://github.com/prometheus-operator/prometheus-operator#quickstart)
+
+Note: the namespace in the ClusterRoleBinding needs to be updated as we're deploying in a namespace other than the default namespace, that's why we have `kind/prometheus-operator-cluster-role-binding.yaml`
```

Comment:
Nitpick: It's a bit weird that you primarily talk about the Prometheus Operator. At first I was confused what component I'm reviewing again :sweat_smile:

Reply:
yes, that's true, now that you said it, it's a bit confusing. I'll update the explanation